project-novica/demo/src/main/java/com/example/demo/controller/AuthenticationController.java

```diff
@@ -0,0 +1,95 @@
+package com.example.demo.controller;
+
+import com.example.demo.model.User;
+import com.example.demo.model.exceptions.EmailAlreadyExistException;
+import com.example.demo.model.exceptions.InvalidArgumentException;
+import com.example.demo.service.AuthenticationService;
+import com.example.demo.util.PasswordHashing;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Controller
+public class AuthenticationController
```

Comment:
Add default mapping for this controller:
Example:
@RequestMapping(value = "/auth")